deployment/distribution/src/main/kotlin/org/codice/ctk/Command.kt

```diff
@@ -56,7 +74,24 @@ fun main(args: Array<String>) {
         Log.logLevel = LogLevel.INFO
     }
 
-    @Suppress("SpreadOperator")
-    org.junit.runner.JUnitCore.main(BASIC_SUITE,
-            *(if (arguments.flag("e")) arrayOf(ERROR_SUITE) else arrayOf()))
+    launchTests(arguments)
+}
+
+@Suppress("SpreadOperator")
+private fun launchTests(arguments: ArgumentCollection) {
+    val request = LauncherDiscoveryRequestBuilder.request().selectors(*BASIC_TESTS).apply {
+        if (arguments.flag("e")) {
+            selectors(*ERROR_TESTS)
+        }
+    }.build()
+
+    val summaryGeneratingListener = SummaryGeneratingListener()
+    LauncherFactory.create().apply {
+        registerTestExecutionListeners(summaryGeneratingListener)
+    }.execute(request)
+
+    PrintWriter(System.out).use {
+        summaryGeneratingListener.summary.printFailuresTo(it)
+        summaryGeneratingListener.summary.printTo(it)
+    }
```

Comment:
❓ Is this adding functionality or just replicating the functionality we used to have?

Reply:
It is the new way of generating the output.